www/admin/account-settings-banner-logging.php

```diff
@@ -164,22 +169,25 @@
                 'type'    => 'checkbox',
                 'name'    => 'logging_proxyLookup',
                 'text'    => $strProxyLookup
-            ),
-/*            array (
-                'type'    => 'checkbox',
-                'name'    => 'logging_sniff',
-                'text'    => $strSniff
-            )*/
+            )
         )
     ),
     array (
         'text'  => $strPreventLogging,
         'items' => array (
             array (
+                'type'    => 'checkbox',
+                'name'    => 'logging_blockInactiveBannerClicks',
+                'text'    => $strBlockInactiveBannerClicks
+            ),
+            array (
+                'type'    => 'break'
+            ),
+            array (
                 'type'    => 'text',
                 'name'    => 'logging_blockAdClicksWindow',
                 'text'    => $strBlockAdClicks,
-                'check'   => 'wholeNumber',
+                'check'   => 'wholeNumber'
```

Comment:
It's good practice to keep the trailing comma in the last multi-line array element. Diffs are more readable in case anything is appended.